modules/ROOT/pages/vm/vm-reference.adoc

```diff
@@ -224,26 +223,26 @@ The flow in which the content is published cannot contain a Listener (`<vm:liste
 `<vm:listener>`
 
 
-A source that creates and listens on VM queues. VM queues are created by placing listeners on them, which is why this listener contains parameters for a queue's behavior, such as it being persistent or not, the max capacity, etc.
+A source that creates and listens on VM queues. VM queues are created by placing listeners on them, which is why this listener contains parameters for a queue's behavior, such as whether or not it is persistent, the max capacity, and so on.
 
 
 ==== Parameters
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Number Of Consumers a| Number |  The number of concurrent consumers to be placed on the queue. As the number of consumers increases, so does the speed at which this source pushes messages into the owning flow. The default is 4 concurrent consumers. |  4 | {nbsp}
+| Configuration | String | The name of the configuration to use. | | *x*
+| Number Of Consumers a| Number |  The number of concurrent consumers to place on the queue. As the number of consumers increases, so does the speed at which this source pushes messages into the owning flow. The default is 4 concurrent consumers. |  4 | 
```

Comment:
This does not make sense to me. I would think it would decrease the speed. Maybe you should double-check this.

Reply:
It actually does make sense to me that messages are pushed at an increased speed with an increase in the number of consumers. Speed would need to increase to keep up with more demand.